src/modules/daemonconfiguration/src/daemonconfiguration.rs

```diff
@@ -113,58 +139,111 @@ impl SystemctlInfo for Systemctl {
         })
     }
 
-    fn show_substate_property(name: &str) -> Result<String, MmiError> {
+    fn show_substate_property(&self, name: &str) -> Result<String> {
         let output = {
             Command::new("systemctl")
                 .args(["show", name, "--property=SubState"])
                 .output()?
         };
         Ok(String::from(std::str::from_utf8(&output.stdout)?))
     }
+
+    fn exists(&self, name: &str) -> Result<bool> {
+        Ok(systemctl_crate::exists(name)?)
+    }
+
+    // Start systemd daemon with name "name". Returns true if successful, false otherwise.
```

Comment:
Minor detail: for comments like this, consider using `///` to serve as documentation. This will "be picked" by developer's editors and displayed nicely when hovering over those methods:

https://doc.rust-lang.org/rust-by-example/meta/doc.html